react/src/objects/centered-column/SprkCenteredColumn.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const SprkCenteredColumn = (props) => {
+  const {
+    children,
+    additionalClasses,
```

Comment:
We'll need idString here too, we don't need  analyticsString though because this isnt a clickable thing.